app/controllers/events_controller.rb

```diff
@@ -69,13 +72,11 @@ def destroy
 
   # GET /events/1/badges
   def badges
-    authorize! :print_badges, @event
```

Comment:
The default authorization check will now check for `badges` instead of `print_badges`. Is this intended?